lib/routes/account.js

```diff
@@ -204,7 +204,8 @@ module.exports = function (
             email: account.email,
             emailCode: account.emailCode,
             emailVerified: account.emailVerified,
-            verifierSetAt: account.verifierSetAt
+            verifierSetAt: account.verifierSetAt,
+            createdAt: parseInt(request.query.createdAt)
```

Comment:
I don't love exposing hooks like this in the public API, but we can probably get away with it in this case.  Some things we could consider to tighten it up:

* Don't respect it when `NODE_ENV` says we're in production mode
* Give it a more private-sounding name like `_createdAt`
* Only allow setting it to dates in the past, not the future

@philbooth thoughts?